docs/resources/azurerm_key_vault.md.erb

```diff
@@ -0,0 +1,109 @@
+---
+title: About the azurerm_key_vault Resource
+platform: azure
+---
+
+# azurerm\_key\_vault
+
+Use the `azurerm_key_vault` InSpec audit resource to test properties and configuration of
+an Azure Key Vault.
+<br />
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+This resource first became available in 1.1.0 of the inspec-azure resource pack.
```

Comment:
Just marking this line so we don't forget to update it. 1.1.0 was already released, so this will be a later version.